File: Codigos estudiantes por lenguaje/Java/Edwin Villarraga/Grafos y Arboles/MatrizAdyacency.java
import java.util.*;
public class Main {
	static int G[][];
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		    int V,E;
		    V=sc.nextInt();
		    E=sc.nextInt();
		    G=new int[V][V];
		    for(int i=0;i<E;++i){
		        int desde,hasta,peso;
		        desde=sc.nextInt();
		        hasta=sc.nextInt();
		        peso=sc.nextInt();
		        G[desde][hasta]=peso;
		        G[hasta][desde]=peso;
		    }
		    for(int i=0;i<V;++i){
		        for(int j=0;j<V;++j){
		        	System.out.print(G[i][j]+"\t");
		        }
		        System.out.println();
		    }
	}
}

File: Codigos estudiantes por lenguaje/Kotlin/Grafos/MatrizAdyacency.kt
import java.util.*
fun main(args: Array<String>) {
    val size=101
    var G= Array(size,{IntArray(size)})
    for(i in 0..G.size-1){
        var colArray=IntArray(size);
        for(j in 0..G.size-1){
            colArray[j]=0
        }
        G[i]=colArray
    }
    val sc=Scanner(System.`in`)
    var V=sc.nextInt()
    val E=sc.nextInt()
    for(i in 0..E-1){
        var desde=sc.nextInt()
        var hasta=sc.nextInt()
        var peso=sc.nextInt()
        G[desde][hasta]=peso
        G[hasta][desde]=peso
    }
    for(i in 0..V-1){
        for(j in 0..V-1){
            print("${G[i][j]}\t")
        }
        print("\n")
    }
}
